### DISCUSSION TITLE
Bootstrap Version

### DISCUSSION THREAD

**Discussion by gravitymedianet** [ericgio/react-bootstrap-typeahead#808] [Q&A] [answered]
Does the current version, 6.2.3 use Bootstrap 5?

**Comment by ericgio** [comment, marked as answer]
Bootstrap 5 is supported, but requires including the BS5-specific styles. Please be sure to read the documentation on [including BS5 CSS](https://github.com/ericgio/react-bootstrap-typeahead#bootstrap-5).
